modules/common/src/main/java/org/eclipse/xpanse/common/security/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.common.security.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.password.Pbkdf2PasswordEncoder;
+
+/**
+ * Spring Security Config.
+ */
+@Configuration
+public class WebSecurityConfig {
```

Comment:
Can we make this class name - SensitiveFieldEncoderConfig?
because we wil have spring web security for other purpose also soon.